editor.planx.uk/src/pages/Teams.tsx

```diff
@@ -72,9 +77,51 @@ const Teams: React.FC<Props> = ({ teams, teamTheme }) => {
     });
   return (
     <Container maxWidth="formWrap">
-      <Typography variant="h2" component="h1" mb={4}>
-        Select a team
-      </Typography>
+      <Box
+        pb={1}
+        sx={{
+          display: "flex",
+          flexDirection: "row",
+          justifyContent: "space-between",
+          alignItems: "center",
+          marginBottom: "40px",
+        }}
+      >
+        <Typography variant="h2" component="h1">
+          Select a team
+        </Typography>
+        {isPlatformAdmin ? (
+          <AddButton
+            onClick={async () => {
+              const newTeamName = prompt("Team name");
+
+              if (newTeamName) {
+                const newSlug = slugify(newTeamName);
+                const teamNameDuplicate = teams.find(
```

Comment:
If there's a slicker way to validate the name, I'd love to get feedback on this

Reply:
I think this is perfect for now! If we had a huge number of teams, or pagination, or users which couldn't view all teams, we'd need something a bit more sophisticated but this works great 👍 

One small nitpick / comment would be that the variable is called `teamNameDuplicate` but it's actually checking the team slug. 

We could also use a guard clause here to avoid conditional nesting, which makes things a little easier to read.

```ts
const isDuplicate = teams.find((team) => team.slug === newSlug);
if (isDuplicate) {
  alert(
    `A team called "${newSlug}" already exists. Enter a unique team name to continue.`
  );
}

// make team and redirect!

```

Reply:
Switched the variable name to include Slug rather than Name, makes more sense for another dev. I have a few other points below on the other parts of the code you mentioned. 

Since they are adding a ``teamName`` I think this should be used in the Alert message rather than the ``newSlug``. Could create a wee bit of confusion with someone thinking "I didn't add that" which would be reduced if we used the name. 

I kept away from using ``isDuplicate`` since the ``find`` method will return an ``object{} || undefined`` rather than a bool. 

Guard clause is a good idea and implemented now! 

Thanks for the feedback